pegasus/sites.v3/code.org/public/learn2016/index.haml

```diff
@@ -13,6 +13,34 @@ social:
   "og:video:type": ""
 ---
 
+:css
+  .fadable-enter {
+    opacity: 0.01;
+  }
+
+  .fadable-enter.fadable-enter-active {
+    opacity: 1;
+    transition: opacity 200ms ease-in;
+  }
+
+  .fadable-appear {
+    opacity: 0.01;
+  }
+
+  .fadable-appear.fadable-appear-active {
+    opacity: 1;
+    transition: opacity 200ms ease-in;
+  }
```

Comment:
Are these class names magic names used by ReactCSSTransitionGroup? Might be worth a comment somewhere. If I were to search for `fadable-appear-active` right now, I wouldn't see it in use anywhere and might think I could delete it from this css file.